fly.toml

```diff
@@ -0,0 +1,36 @@
+# fly.toml app configuration file generated for diveharder-api-py on 2024-04-01T19:50:43-04:00
+#
+# See https://fly.io/docs/reference/configuration/ for information about how to use this file.
+#
+
+app = 'diveharder-api-py'
+primary_region = 'iad'
+kill_signal = 'SIGTERM'
+
+[build]
+  dockerfile = 'Dockerfile'
+  ignorefile = '.dockerignore'
+
+[http_service]
+  internal_port = 1234
+  force_https = true
+  auto_stop_machines = true
+  auto_start_machines = true
+  min_machines_running = 1
```

Comment:
I suggest adding a `max_machines_running` to prevent accidental scaling when experiencing load, as this could lead to costs